src/Simplex/Chat/Controller.hs

```diff
@@ -282,6 +282,10 @@ data ChatCommand
   | APIGroupLinkMemberRole GroupId GroupMemberRole
   | APIDeleteGroupLink GroupId
   | APIGetGroupLink GroupId
+  | APICreateMemberContact GroupId GroupMemberId
+  | APISendMemberContactInvitation {contactId :: ContactId, msgContent_ :: Maybe MsgContent}
+  | CreateMemberContact GroupName ContactName
+  | SendMemberContactInvitation {contactName :: ContactName, message_ :: Maybe Text}
```

Comment:
I don't think the terminal commands are needed here, it's not too useful. It's actually the argument to modify send to allow sending to members, or you'll have to put some logic in terminal send to do two things. Member have a unique local display name, so we need to preserve the ability to simply send messages to them. And if we do, we may as well create contact and send message in one go from the UI as well (although it's a separate consideration).